modules/ROOT/pages/analytics-chart.adoc

```diff
@@ -38,4 +38,4 @@ image::analytics-performance.png[Avg Latency 2.74k]
 
 Enable or disable the tracking of policy violations by checking the Include Policy Violations checkbox on an Analytics chart screen. The chart will show traffic rejected, or not rejected, by a policy.
 
-image::policy-violations-dashboard.png[]
\ No newline at end of file
+image::policy-violations-dashboard.png[]
```

Comment:
If this image is described in the paragraph on line 39, I think we need to reference that--something like... "The chart shows traffic that is rejected or not rejected by a policy, as shown in the following image:"